src/common/types/contentPreview.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+
+import ContentPreview from '../../elements/content-preview';
```

Comment:
We typically want to avoid importing element-specific code into common/component modules.

Reply:
Moved the type definition to features/message-preview-content, so that's not an issue anymore, thank you!